src/ape_test/fixtures.py

```diff
@@ -12,23 +12,23 @@
 class PytestApeFixtures:
```

Comment:
something evil maybe lol:
```suggestion
class PytestApeFixtures(ManagerAccessBase):
```